Added contributor list format and sorting instructions

stevepiercy opened the pull request:
Added contributor list format and sorting instructions as a comment

Closes #1023 

<!-- readthedocs-preview icalendar start -->
----
📚 Documentation preview 📚: https://icalendar--1022.org.readthedocs.build/

<!-- readthedocs-preview icalendar end -->

stevepiercy commented on docs/contribute/credits.rst:
```diff
@@ -30,6 +30,7 @@ Original author
 
 Contributors
 ------------
+.. Sort alphabetically and use template: `your name <https://github.com/github_username>`_
```
```suggestion
.. Sort alphabetically and use the following template.
.. `your name <https://github.com/github_username>`_
.. Don't disclose your email address to spam harvesters.
```

stevepiercy commented on docs/contribute/credits.rst:
```diff
@@ -113,8 +115,9 @@ Contributors
 - Wichert Akkerman <wichert@wiggy.net>
-- Howard Liao <howard940114@gmail.com>
 - Zepyoor Khechadoorian <zepyoorkhechadoorian@gmail.com>
+.. Sort alphabetically and use template: `your name <https://github.com/github_username>`_
```
```suggestion
.. Sort alphabetically and use the following template.
.. `your name <https://github.com/github_username>`_
.. Don't disclose your email address to spam harvesters.
```

SashankBhamidi commented on docs/contribute/credits.rst:
```diff
@@ -114,8 +118,11 @@ Contributors
+.. Sort alphabetically and use the following template.
+.. `your name <https://github.com/github_username>`_
+.. Don't disclose your email address to spam harvesters.
+
```
The redundancy required? I feel like keeping it in the top of the file is enough

stevepiercy commented:
@SashankBhamidi I'll let you handle auto-merge or not, since I updated the repo config to handle it better. I think it's fine to auto-merge this one.

SashankBhamidi commented:
@stevepiercy Does this close #1024?

stevepiercy commented:
PRs should not close higher number issues or PRs. I put that in there for a second, then thought, nah, @SashankBhamidi is playing around.

stevepiercy commented:
I removed it from the required checks. I thought it was the link checker when I added it, but it's RTD creating a link in the description of the PR to the preview of the built docs.
